### PR TITLE
Added german readme translation

### PR DESCRIPTION
**Pull request by 0xGhostwire**
I added the german readme translation.

### CODE REVIEW

File: de_DE-README.md [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+- Es dürfen keine öffentlichen Massenserver gehostet werden. Der Sinn von Fika ist es COOP mit Freunden spielen zu können.
+- Es ist nicht erlaubt Fikas Code zu kopieren oder zu replizieren. Es ist zudem nicht gestattet die von den Entwicklern und Künstlern erstellten Assets zu verwenden.
+
+## Voraussetzungen
+
+Fika benötigt generelles Verständnis von Computern, Netzwerken und Aki. Wenn du nicht mit diesen Konzepten vertraut bist ist dieses Projekt eher nichts für dich. Wir bitten dich um Verständnis und diese Entscheidung zu aktzeptieren.
+
+### Hosten
+
+- Router und Internetprovider mit Unterstützung für **Portweiterleitung** oder **UPnP**
```
[the remark this answers is not in the document]
Comment (Lacyway) [reply de_DE-README.md:58]: @0xGhostwire is this something that could be changed? I don't speak German my self so unsure if you're in agreement with @maxomatic458

[the remark this answers is not in the document]
Comment (maxomatic458) [reply de_DE-README.md:58]: @Lacyway i made a PR on @0xGhostwire's branch to fix some mistakes

**Comment by 0xGhostwire** [comment]
@Lacyway, @maxomatic458 changes made sense I merged his commits into my branch the PR should be updated now

**Comment by Lacyway** [comment]
Thanks to both of you!

**Pull request merged**